packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -790,6 +789,14 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @protected
   late final ScrollbarPainter scrollbarPainter;
 
+  /// Overridable getter to indicate that the scrollbar should be visible, even
+  /// when a scroll is not underway.
+  ///
+  /// Subclasses can override this getter to incorporate theme defaults, like the
```

Comment:
Subclasses can override this getter to make its value depend on an inherited theme.